GVFS/GVFS/CommandLine/SparseVerb.cs

```diff
@@ -120,6 +120,7 @@ protected override void Execute(GVFSEnlistment enlistment)
                     {
                         if (directories.Count > 0)
                         {
+                            this.Output.WriteLine("Removing all folders from sparse list. When the sparse list is empty, all folders are projected.");

```

Comment:
Use `this.WriteMessage` here?  It looks like we're writing the other messages in this method to the log.

A bit outside the scope of this PR, but are the values of the parameters being written to the log somewhere (I had trouble finding where they would be)?  If they're not, could we log them?  If we have support issues it would be helpful to see what parameters were passed to the verb.